tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.robot

```diff
@@ -128,6 +128,9 @@ Create VCH - defaults
     Log To Console  Installer completed successfully: %{VCH-NAME}
 
     Run Regression Tests
+
+    ${sessions}=  Check vSphere Sessions  "vic-machine"
+    Should Be Equal As Integers  ${sessions}  0
```

Comment:
I started refactoring this before I saw your comment because I was actually unhappy with the way I had this designed. I moved this check into a couple of keywords so that this check runs each time we do `vic-machine create` (I'm still running tests and making sure stuff works how I want) but I could, instead, do a single test that runs all of the vic-machine subcommands in Regression Tests, since maybe adding it to the keywords in the current version is overkill?.. I've been torn all day about which approach is better.